cmd/dep/root_analyzer.go

```diff
@@ -53,6 +63,58 @@ func (a *rootAnalyzer) InitializeRootManifestAndLock(dir string, pr gps.ProjectR
 	return
 }
 
+func (a *rootAnalyzer) cacheDeps(pr gps.ProjectRoot) error {
+	logger := a.ctx.Err
+	g, _ := errgroup.WithContext(context.TODO())
+	concurrency := 4
+
+	syncDep := func(pr gps.ProjectRoot, sm gps.SourceManager) error {
+		if err := sm.SyncSourceFor(gps.ProjectIdentifier{ProjectRoot: pr}); err != nil {
+			logger.Printf("Unable to cache %s - %s", pr, err)
+			return err
+		}
+		logger.Printf("Cached %s", pr)
+		return nil
+	}
+
+	deps := make(chan gps.ProjectRoot)
+
+	for i := 0; i < concurrency; i++ {
+		g.Go(func() error {
+			for d := range deps {
+				err := syncDep(d, a.sm)
+				if err != nil {
+					return err
+				}
+			}
+			return nil
+		})
+	}
+
+	g.Go(func() error {
+		defer close(deps)
+		for ip := range a.directDeps {
+			logger.Printf("Package %q, analyzing...", ip)
```

Comment:
Let's change this to `Caching package %q` because we aren't actually analyzing anything.